commit 8f00911a21f4e95de84c60e09cc4df173e5b6701
Author: Theodore Ts'o <tytso@mit.edu>
Date:   Tue Jun 12 18:25:34 2012 -0400

    libquota: remove unneeded #include of <sys/quota.h>
    
    The attempted inclusion of sys/quota.h is causing failures in when
    building on the hurd and freebsd platforms for Debian.  It's not
    necessary any more, so just remove the #include.
    
    Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>

diff --git a/lib/quota/quotaio.c b/lib/quota/quotaio.c
index 67e74fc..c0ebaa1 100644
--- a/lib/quota/quotaio.c
+++ b/lib/quota/quotaio.c
@@ -15,7 +15,6 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/file.h>
-#include <sys/quota.h>
 
 #include "common.h"
 #include "quotaio.h"
